gravitee-am-ui/src/app/domain/applications/application/overview/overview.component.html

```diff
@@ -91,37 +91,14 @@ <h2>Initiate the Login Flow</h2>
             <p><b>Note:</b> remember to adapt the request to use the client secret you copied during the application creation</p>
           </blockquote>
           <div class="code">
-            <button
-              mat-icon-button
-              matSuffix
-              matTooltip="Copy to clipboard"
-              (click)="copyToClipboard(pre_authorization_code_exchange, authorizationHeader)"
-            >
+            <button mat-icon-button matSuffix matTooltip="Copy to clipboard" (click)="copyToClipboard(pre_authorization_code_exchange)">
               <mat-icon>content_copy</mat-icon>
             </button>
-            <button
-              *ngIf="!isHidden(safeAuthorizationHeader)"
-              mat-icon-button
-              matSuffix
-              matTooltip="Hide sensitive data"
-              (click)="safeAuthorizationHeader = hidden"
-            >
-              <mat-icon>visibility_off</mat-icon>
-            </button>
-            <button
-              *ngIf="isHidden(safeAuthorizationHeader)"
-              mat-icon-button
-              matSuffix
-              matTooltip="Reveal sensitive data"
-              (click)="safeAuthorizationHeader = authorizationHeader"
-            >
-              <mat-icon>visibility</mat-icon>
-            </button>
 
             <pre #pre_authorization_code_exchange class="multiline">
 curl -X POST \
   {{ baseUrl }}/oauth/token \
-  -H 'Authorization: Basic {{ safeAuthorizationHeader }}' \
+  -u '{{ clientId }}:&lt;add you secret here&gt;' \
```

Comment:
thought: there is no info that it has to be encoded with base64. Would it be better to change it to "******" or "access_token" and then describe what it means above in the text? 

Regarding this line, 249 and 161

Reply:
the '-u' option will encode in Base64. I will update the "add your secret here" with a "<_client_secret_>"